apps/web/src/components/settings/AccountSettings.tsx

```diff
@@ -51,29 +73,115 @@ export default function AccountSettings({ user }: AccountSettingsProps) {
 							value={newFirstName}
 							onChange={(e) => setNewFirstName(e.target.value)}
 						/>
+						{!newFirstName ? (
+							<div className={"mt-1 text-sm text-red-500"}>
+								This field can't be empty!
+							</div>
+						) : null}
 					</div>
 					<div>
-						<Label htmlFor="lastname">Last Name</Label>
+						<Label htmlFor={"lastname"}>Last Name</Label>
 						<Input
 							className="mt-2"
 							name="lastname"
 							value={newLastName}
 							onChange={(e) => setNewLastName(e.target.value)}
 						/>
+						{!newLastName ? (
+							<div className={"mt-1 text-sm text-red-500"}>
+								This field can't be empty!
+							</div>
+						) : null}
 					</div>
-					<Button
-						className="mt-5"
-						onClick={() => {
-							toast.loading("Updating name...");
-							runModifyAccountSettings({
-								firstName: newFirstName,
-								lastName: newLastName,
-							});
-						}}
+					{/*<div className={"col-span-full"}>*/}
```

Comment:
question: Will this code be used?

Reply:
This is the email placeholder. We said we didn't want to be able to change email right now, but I believe it's something we are looking at in the future once we get db refactors out and do some more work with the auth.